docs/IE_DG/Paddle_Support.md

```diff
@@ -0,0 +1,34 @@
+# Paddle Support in the OpenVINO™ {#openvino_docs_IE_DG_Paddle_Support}
+
+Starting from the 2022.1 release, OpenVINO™ supports reading native Paddle models.
+`Core::ReadNetwork()` method provides a uniform way to read models from IR or Paddle format, it is a recommended approach to reading models.
+
+## Read Paddle Models from IR
+
+After [Convert a Paddle Model](../MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md) to [Intermediate Representation (IR)](../MO_DG/IR_and_opsets.md), it can be read as recommended. Example:
```

Comment:
```suggestion
After [Converting a Paddle Model](../MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md) to [Intermediate Representation (IR)](../MO_DG/IR_and_opsets.md), it can be read as recommended. Example:
```

Reply:
Let's add one small fix before merging